### PR TITLE
add django wagtail feeds

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by helloreuben** [comment]
@sannykr needs wagtail to be built and available first

**Comment by sannykr** [comment]
@helloreuben : Thanks. wagtail failed because of lack of dependencies. Once its dependencies are built, then only wagtail will build and then I will work on this package.

**Comment by sannykr** [comment]
@jakirkham : Could you please review and merge if everything looks okay

**Comment by sannykr** [comment]
@isuruf : Should it be noarch?

**Comment by isuruf** [comment]
Looks like it can be.

**Comment by sannykr** [comment]
@jakirkham : Could you please review and merge if everything looks okay.

**Comment by jakirkham** [comment]
Sorry I don't have time to look ATM. Could you please ping `@conda-forge/staged-recipes`, @sannykr? That is a larger group of people who can take a look and merge. So should give you a better response time than pinging particular individuals. Thanks.

**Comment by sannykr** [comment]
@jakirkham : Thanks

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : could you please review and merge if everything looks okay.

File: recipes/django-wagtail-feeds/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set name = "django-wagtail-feeds" %}
+{% set version = "0.0.8" %}
+{% set sha256 = "e75237237b7dc698a80c2a2826a8fdd92c60e070835ed2f24b1059a553b46c9f" %}
+
+
```
Comment (juoceano) [inline recipes/django-wagtail-feeds/meta.yaml]: You can remove the extra space later.

File: recipes/django-wagtail-feeds/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (juoceano) [inline recipes/django-wagtail-feeds/meta.yaml]: No need for the `fn`. When absent it will come from the URL.

**Comment by ocefpaf** [comment]
Thanks @juoceano!

**Pull request merged**